code/datums/elements/pet_bonus.dm

```diff
@@ -35,5 +39,8 @@
 	new /obj/effect/temp_visual/heart(pet.loc)
 	SEND_SIGNAL(pet, COMSIG_ANIMAL_PET, petter, modifiers)
 	if(emote_message && prob(33))
-		pet.manual_emote(emote_message)
+		if(emote_message)
+			pet.manual_emote(emote_message)
+		playsound(pet, emote_sound, 120, TRUE, TRUE, FALSE)
+		INVOKE_ASYNC(pet, TYPE_PROC_REF(/mob, emote), emote)
```

Comment:
nitpick, maybe want to rename `emote` to `emote_type` so this is easier to understand.

Reply:
renamed it to `emote_name` because `emote_type` already exists on emotes